app/lib/tool/backfill/backfill_new_fields.dart

```diff
@@ -12,5 +15,16 @@ final _logger = Logger('backfill_new_fields');
 /// CHANGELOG.md must be updated with the new fields, and the next
 /// release could remove the backfill from here.
 Future<void> backfillNewFields() async {
-  _logger.info('No backfill needed.');
+  _logger.info('Backfilling PackageState...');
+  final query = dbService.query<PackageState>();
```

Comment:
Why not just avoid falling back to older runtime versions.

This won't fix it, new PackageState will be created when new packages are published.

And the new runtimeVersion will trigger all new PackageState entities for the next deployment we make.

Reply:
Good point, removing it.